drivers/pds/lib/utils.go

```diff
@@ -1244,7 +1244,7 @@ func SetupPDSTest(ControlPlaneURL, ClusterType, AccountName, TenantName, Project
 	return tenantID, dnsZone, projectID, serviceType, clusterID, err
 }
 
-//RegisterClusterToControlPlane checks and registers the given target cluster to the controlplane
+// RegisterClusterToControlPlane checks and registers the given target cluster to the controlplane
```

Comment:
I believe this is added when you run Makefile.

Reply:
Yes. Go formatting was missed by someone who edited this file in previous PRs